nodejs/week1/avg.js

```diff
@@ -0,0 +1,12 @@
+// Excluding "node" and script name
+const args = process.argv.slice(2);
```

Comment:
Very nice to see that you extract a confusing variable `process.argv` into a variable that is easier to understand. 🤩